CHANGELOG.md

```diff
@@ -5,6 +5,27 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v0.46.0 (2025-05-06)
+
+### ✨ New
+
+- [#2162](https://github.com/meltano/sdk/issues/2162) Expose metric dictionary to make logging metrics as JSON easier
+- [#2977](https://github.com/meltano/sdk/issues/2977) Limit extraction to 25 records by default in builtin tap tests
+- [#2989](https://github.com/meltano/sdk/issues/2989) Allow pagination to continue after an empty page
+
+### 🐛 Fixes
+
+- [#3020](https://github.com/meltano/sdk/issues/3020) In SQL targets, use stream name as table name instead of splitting by hyphen if `default_target_schema` is speficied
```

Comment:
```suggestion
- [#3020](https://github.com/meltano/sdk/issues/3020) In SQL targets, use stream name as table name instead of splitting by hyphen if `default_target_schema` is specified
```